graphql/schema.graphql

```diff
@@ -219,6 +219,7 @@ type Version {
   activated: Boolean
   taskStatusCounts(options: BuildVariantOptions): [StatusCount!]
   buildVariants(options: BuildVariantOptions): [GroupedBuildVariant]
+  buildVariantStats(options: BuildVariantOptions): [GroupedStatusCount!]
```

Comment:
Would it maybe be possible to add a graphql test for this so that we can see what the graphql query and results would look like?